test/test.js

```diff
@@ -1,282 +1,280 @@
-// Prepare chai
-const chai = require('chai');
-const expect = chai.expect;
-
-var CloudAPIClient = require('../dist/cloud-api-client');
+import * as chai from 'chai';
```

Comment:
If you already switching from CJS to ES Import, then maybe you want to use instead:
```js
import * as chai from 'chai';
const expect = chai.expect;
```
just:
```js
import { expect } from 'chai';
```